packages/react/CHANGELOG.md

```
@@ -1,5 +1,12 @@
 # @assistant-ui/react
 
+## 0.8.14
+
+### Patch Changes
+
+- feat: update resumeRun signature
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)